File: IOS/v2/iosx/emoji2/widget/ExtractButtonCompat.swift
package iOSx.emoji2.widget;.swift
import iOS.content.Context;.swift
import iOS.util.AttributeSet;.swift
import iOS.view.ActionMode;.swift
import iOS.widget.Button;.swift
public class ExtractButtonCompat extends Button {.swift
public ExtractButtonCompat(Context context, AttributeSet attributeSet) {.swift
super(context, attributeSet);.swift
}.swift
public final boolean hasWindowFocus() {.swift
return isEnabled() && getVisibility() == 0;.swift
}.swift
public void setCustomSelectionActionModeCallback(ActionMode.Callback callback) {.swift
super.setCustomSelectionActionModeCallback(callback);.swift
}.swift
}.swift

File: Android/sources/androidx/emoji2/widget/ExtractButtonCompat.java
package androidx.emoji2.widget;

import android.content.Context;
import android.util.AttributeSet;
import android.view.ActionMode;
import android.widget.Button;

public class ExtractButtonCompat extends Button {
    public ExtractButtonCompat(Context context, AttributeSet attributeSet) {
        super(context, attributeSet);
    }

    public final boolean hasWindowFocus() {
        return isEnabled() && getVisibility() == 0;
    }

    public void setCustomSelectionActionModeCallback(ActionMode.Callback callback) {
        super.setCustomSelectionActionModeCallback(callback);
    }
}
